Cargo.toml

```diff
@@ -18,11 +18,11 @@ committable = "0.2"
 derivative = "2.2"
 futures = "0.3"
 hex = "0.4.3"
-hotshot = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.69" }
-hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.69" }
-hotshot-events-service = { git = "https://github.com/EspressoSystems/hotshot-events-service.git", tag = "0.1.40" }
-hotshot-task-impls = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.69" }
-hotshot-types = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.69" }
+hotshot = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.70" }
+hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.70" }
+hotshot-events-service = { git = "https://github.com/EspressoSystems/hotshot-events-service.git", branch = "update-hotshot" }
```

Comment:
shouldn't the event service be  merged first so that we can point to a new tag

Reply:
Ooops. I'll do another PR

Reply:
https://github.com/EspressoSystems/marketplace-builder-core/pull/66